oipip-0004.md

```diff
@@ -0,0 +1,37 @@
+# OIPIP-0004 : Change name of OIP Object from artifact to record
+
+```
+Number:  OIPIP-0004
+Title:   Change from artifact to record
+Type:    Standard
+Status:  Draft
+Authors: Sky Young <sky@alexandria.io>
+Created: 2018-05-30
+```
+
+## Abstract
+
+This OIPIP proposes to change the name of the data JSON Object in OIP from `artifact` to `record`.
+
+## Motivation
+
+Artifact can be misinterpreted. The second definition in the Merriam-Webster dictionary is:
+
+```
+2.a : a product of artificial character (as in a scientific test) due usually to extraneous (such as human) agency
+2.c : a defect in an image (such as a digital photograph) that appears as a result of the technology and methods used to create and process the image 
+```
+
+This is particularly bad for scientific use of OIP.
+
```

Comment:
Also explain why *record* is preferred. Something like common definitions include the idea of something that registers something like evidence into a permanent medium.

Reply:
Done